src/test/generate-matrix.py

```diff
@@ -210,6 +210,7 @@ def __str__(self):
     env=dict(
         TEST_INSTALL="t",
     ),
+    platform="linux/amd64",
```

Comment:
Is this a stray change? If it is supposed to be part of this commit it might be useful to explain why in the commit message (it isn't clear to me on first view anyway)